lib/permutations.rb

```diff
@@ -1,4 +1,14 @@
 
 def permutations?(string1, string2)
-  raise NotImplementedError, "permutations? not implemented"
+  return false if string1.length != string2.length
+
+  string1.chars.each do |char|
+    if string2.include?(char)
```

Comment:
`.include?` is an O(n) operation, making this an O(n^2) method.  Can you think of a better way to do this with a hash?